pyserini/prebuilt_index_info.py

```diff
@@ -284,71 +284,71 @@
     "msmarco-v2-doc": {
         "description": "Lucene index of the MS MARCO V2 document corpus.",
-        "filename": "lucene-index.msmarco-v2-doc.20220808.4d6d2a.tar.gz",
-        "readme": "lucene-index.msmarco-v2-doc.20220808.4d6d2a.README.md",
+        "filename": "lucene-inverted.msmarco-v2-doc.20220808.4d6d2a.tar.gz",
+        "readme": "lucene-inverted.msmarco-v2-doc.20220808.4d6d2a.README.md",
         "urls": [
-            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.msmarco-v2-doc.20220808.4d6d2a.tar.gz",
+            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene/lucene-inverted.msmarco-v2-doc.20220808.4d6d2a.tar.gz",
         ],
-        "md5": "0599bd6ed5ee28390b279eb398ef0267",
-        "size compressed (bytes)": 63431299815,
+        "md5": "11017a76d2927294977c360a174ba1be",
+        "size compressed (bytes)": 63431305265,
         "total_terms": 14165667143,
         "documents": 11959635,
         "unique_terms": 44860768,
         "downloaded": False
     },
     "msmarco-v2-doc-slim": {
         "description": "Lucene index of the MS MARCO V2 document corpus ('slim' version).",
-        "filename": "lucene-index.msmarco-v2-doc-slim.20220808.4d6d2a.tar.gz",
-        "readme": "lucene-index.msmarco-v2-doc.20220808.4d6d2a.README.md",
+        "filename": "lucene-inverted.msmarco-v2-doc-slim.20220808.4d6d2a.tar.gz",
+        "readme": "lucene-inverted.msmarco-v2-doc.20220808.4d6d2a.README.md",
         "urls": [
-            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.msmarco-v2-doc-slim.20220808.4d6d2a.tar.gz",
+            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene/lucene-inverted.msmarco-v2-doc-slim.20220808.4d6d2a.tar.gz",
         ],
-        "md5": "4dfc5549e3c15abec4b9694542a376d1",
-        "size compressed (bytes)": 7172175394,
+        "md5": "39272218094950979303f17e7202d636",
+        "size compressed (bytes)": 7172182321,
         "total_terms": 14165667143,
         "documents": 11959635,
         "unique_terms": 44860768,
         "downloaded": False
     },
     "msmarco-v2-doc-full": {
         "description": "Lucene index of the MS MARCO V2 document corpus ('full' version).",
-        "filename": "lucene-index.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
-        "readme": "lucene-index.msmarco-v2-doc.20220808.4d6d2a.README.md",
+        "filename": "lucene-inverted.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
+        "readme": "lucene-inverted.msmarco-v2-doc.20220808.4d6d2a.README.md",
         "urls": [
-            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
+            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene/lucene-inverted.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
         ],
-        "md5": "fc6f546898725617eb5ca7a144bef531",
-        "size compressed (bytes)": 119537276117,
+        "md5": "66a9da92766dc7ee6eec168adf242a66",
+        "size compressed (bytes)": 119537262576,
```

Comment:
I thought the diffs are name changes only, why is the size different?

Reply:
I unpacked, renamed, and repacked -> the name is part of the tarball, I think - so size changes slightly.